dev/data/project.json

```diff
@@ -10,5 +10,11 @@
     "metadata": {
       "name": "IncredibleProject"
     }
+  },
+  {
+    "kind": "Project",
+    "metadata": {
+      "name": "testing"
```

Comment:
Named it "testing" instead of "end-to-end testing" or something like that because projects are listed alphabetically and I wanted the "perses" project to still be first.